app/src/token_info.rs

```diff
@@ -0,0 +1,31 @@
+// An untrusted token information list provided by penumbra team
+// https://github.com/iron-fish/verified-assets/blob/main/mainnet.json
+// We were told to used this in the meantime while a trusted token metadata
+// service is being developed
+pub static TOKEN_LIST: &str = r#"
+{
+    "schemaVersion": 2,
+    "assets": [
+        {
+            "identifier": "51f33a2f14f92735e562dc658a5639279ddca3d5079a6d1242b2a588a9cbf44c",
+            "symbol": "IRON",
+            "decimals": 8,
+            "logoURI": "https://ironfish.network/favicon.ico",
+            "website": "https://ironfish.network"
```

Comment:
Do we really need these values? Are we going to use them somehow? If not, maybe we can save some space